ci/build.bat

```diff
@@ -56,14 +56,18 @@ bin\nvim --version || goto :error
 :: Functional tests
 mingw32-make functionaltest VERBOSE=1 || goto :error
 
+if defined USE_GCOV (
+  C:\msys64\usr\bin\bash -lc "cd /c/projects/neovim; bash <(curl -s https://codecov.io/bash) -c -F functionaltest || echo 'codecov upload failed.'"
+)
+
 :: Old tests
 setlocal
 set PATH=%PATH%;C:\msys64\usr\bin
 mingw32-make -C "%~dp0\..\src\nvim\testdir" VERBOSE=1
 endlocal
 
 if defined USE_GCOV (
-  C:\msys64\usr\bin\bash -lc "cd /c/projects/neovim; bash <(curl -s https://codecov.io/bash) || echo 'codecov upload failed.'"
+  C:\msys64\usr\bin\bash -lc "cd /c/projects/neovim; bash <(curl -s https://codecov.io/bash) -c -F oldtest || echo 'codecov upload failed.'"
```

Comment:
Downloading the same script as above. A bit icky, but I guess saving and managing files is more icky.

Reply:
> I guess saving and managing files is more icky.

That was my thought.  The reporting is purely optimistic, so failures to download the file won't cause the build to fail.